src/utils.js

```diff
@@ -2,24 +2,29 @@ import parse from "url-parse";
 
 export const defaultPortalUrl = "https://siasky.net";
 
-export function open(portalUrl, skylink) {
-  const url = getUrl(portalUrl, parseSkylink(skylink));
+export const options = {
```

Comment:
I don't know about `Global` since the options can be different for different function calls, e.g. you can try to upload to two different portals simultaneously. I'll call it `defaultOptions`